src/test/java/redis/clients/jedis/tests/JedisPoolTest.java

```diff
@@ -149,10 +149,13 @@ public void startWithUrlString() {
     j.auth("foobared");
     j.select(2);
     j.set("foo", "bar");
+    j.close();
+
     JedisPool pool = new JedisPool("redis://:foobared@localhost:6380/2");
     Jedis jedis = pool.getResource();
     assertEquals("PONG", jedis.ping());
     assertEquals("bar", jedis.get("foo"));
+    jedis.close();
```

Comment:
why not using try resource block?

Reply:
just finding out the *closeable* Jedis(es) first, before using try-resource.

Reply:
@sazzad16 I'm not sure I understand the answer

Reply:
@gkorland In this PR, I tried only to find out unclosed objects and `close()` them. There are too many places where try-with-resources can be used which can be done in separate PR. IMHO, using those here and there, in a few random places, would not fulfil our willingness of using try-with-resources. Nevertheless, changes in this PR will also help us to find out the places to put try-with-resources.